azure-pipelines.yml

```diff
@@ -162,6 +162,56 @@ jobs:
               /p:TeamName=DotNetCore
               /p:OfficialBuildId=$(BUILD.BUILDNUMBER)
             displayName: Sign packages
+    - job: Run_SDL
+      displayName: Run SDL tool
+      variables:
+        - group: SDL_Eng_KeyVault
+      dependsOn:
+        - Windows_NT
+        - Linux
+        - Validate_Helix
+        - Validate_Signing
+        - Asset_Registry_Publish
+      steps:
+        - checkout: self
+          clean: true
+        - powershell: eng\common\build.ps1
```

Comment:
If you're doing a build, you don't need to have this depend on any of the other builds. This can run independently.

Reply:
Agreed, fixed.